__tests__/endpoints/topics.test.js

```diff
@@ -6,19 +6,11 @@ afterAll(() => db.end());
 describe("/api/topics", () => {
 	test("GET: 200 sends an array of topics to the client", () => {
 		return request(app)
-			.get("/api/topics?sort")
+			.get("/api/topics")
 			.expect(200)
 			.then(({ body: { topics } }) => {
 				expect(topics.length).toBe(3);
-				topics.forEach((topic) => {
-					expect(typeof topic.slug).toBe("string");
-					expect(typeof topic.description).toBe("string");
-				});
+				expect(topics).toMatchObject(data.topicData);
```

Comment:
Knowing that we are aiming for confidence with testing, referring to the imported data is a bit iffy as you're referring to something stored outside of the scope of your current test block, but also some original data (from _before_ seeding) that may not be reflective of your actual database response (will definitely not work anywhere else). It is also not clear at first where data is referring to. You want to expect that each topic has a key of description and slug - so I would strongly recommend that you refactor to do that explicitly.